database/engine/rrdengineapi.c

```diff
@@ -1034,3 +1034,264 @@ void rrdeng_prepare_exit(struct rrdengine_instance *ctx)
     //metalog_prepare_exit(ctx->metalog_ctx);
 }
 
+int rrdeng_store_past_metrics_page_init(RRDDIM_PAST_DATA *dim_past_data, REPLICATION_STATE *rep_state){
+
+    RRDSET *st = rrdset_find_byname(rep_state->host, dim_past_data->rrdset_id);
+    if(unlikely(!st)) {
+        error("%s: Abort Replication - Cannot find chart with name_id '%s' on host '%s'.", REPLICATION_MSG, dim_past_data->rrdset_id, rep_state->host->hostname);
+        return 1;
+    }
+    dim_past_data->rd = rrddim_find(st, dim_past_data->rrddim_id);
+    if(unlikely(!dim_past_data->rd)) {
+        error("%s: Abort Replication - Cannot find dimension with id '%s' in chart '%s' on host '%s'.", REPLICATION_MSG, dim_past_data->rrddim_id, dim_past_data->rrdset_id, rep_state->host->hostname);
+        return 1;
+    }
+
+    /* Create a page */
+    // struct rrdeng_collect_handle *rep_handle;
+    struct pg_cache_page_index *page_index;
+    struct rrdengine_instance *ctx;
+    RRDDIM *rd = dim_past_data->rd;
+
+    // rep_handle = &rd->state->handle.rrdeng;
+    ctx = get_rrdeng_ctx_from_host(rd->rrdset->rrdhost);
+    dim_past_data->ctx = ctx;
+    page_index = rd->state->page_index;
+
+    // rep_handle->prev_descr = NULL;
+    // rep_handle->unaligned_page = 0;
```

Comment:
This function has commented code, is there a special motive to keep them instead to remove?